src/core/lib/channel/channel_stack.h

```diff
@@ -67,6 +69,13 @@ typedef struct {
   grpc_call_context_element *context;
 } grpc_call_element_args;
 
+typedef struct {
+  uint64_t bytes_in;
```

Comment:
These two fields should be exactly grpc_transport_stream_stats (recommend including that struct within this one).